### DISCUSSION TITLE
Add `rel="me"` to profile website link

### DISCUSSION THREAD

**Discussion by smichaelsen** [monkeytypegame/monkeytype#4014] [Ideas]
Semantically it would be correct to give the website link on the profile page the `rel="me"` attribute.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/me

Real life benefit: People could verify her Monkeytype profiles on Mastodon: https://mastodon.online/@smic

I took the freedom to create a PR: https://github.com/monkeytypegame/monkeytype/pull/4015

**Comment by ghost** [comment]
Completed with #4015
